src/components/FileList/FileElement.js

```diff
@@ -14,6 +14,32 @@ import {
   lengthLimit,
 } from '../../common/fileUtils';
 
+
+const useRecentlyUpdated = (latestDate, fileId) => {
+  const [justUpdated, setJustUpdated] = useState(false);
+
+  useEffect(() => {
+    let FIVE_MIN = 300000;
+    let docs = sessionStorage.getItem('kf_udpated_docs')
```

Comment:
why `kf_updated` ?  I was thinking that appending `_docs` to it could keep from naming collisions in case we want to use the same mechanism to show rows of newly added documents as well.

Reply:
'updated' is misspelled.